apps/infrastructure/http/flask_app.py

```
@@ -0,0 +1,122 @@
+
+        body = request.get_json(silent=True) or {}
+        try:
+            customer = CustomerIn(
+                user_id=body["customer"]["user_id"],
+                email=body["customer"].get("email"),
+            )
+            items = [
+                OrderItemIn(sku=i["sku"], quantity=int(i["quantity"]))
+                for i in body["items"]
+            ]
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)